docs/simple_switch.md

```diff
@@ -685,3 +685,12 @@ you have two actions for table `t` where one has the action
 `m.read(result_field2)`, or if both of those calls are in the same
 action.  All calls to the `read()` method for `m` must have the same
 result parameter where the result is written.
+
+### BMv2 `meter rate and burst_size units` implementation notes.
+### Units help configuring meter with simple_switch_CLI
```

Comment:
This is not specific to simple_switch so I don't understand why we are putting this in this document.
Why not start a separate runtime_CLI documentation if this is your intent?

Reply:
On 04/10, in the P4 Slack channel, a note said to add the info to simple_switch.md.  The note is copied below.

"If you learn the answer, it would be good to document the answer in this file: https://github.com/p4lang/behavioral-model/blob/master/docs/simple_switch.md"

After all, p4runtime_CLI does not run unless simple_switch is running first.  

Anyway, your point makes sense.  Let me add a new file and move the documentation there.  thanks.

Reply:
> After all, p4runtime_CLI does not run unless simple_switch is running first.

That is not accurate. p4runtime_CLI works with any bmv2 target. It does not have to be simple_switch.

Reply:
I see - thanks.  Sorry, I haven't used other bmv2 targets and have used p4runtime_CLI only with simple_switch.  If I do not launch the simple_switch, I see this error.

```
hemant@ubuntu:~/behavioral-model/tools$ ./runtime_CLI.py
Could not connect to thrift client on port 9090
Make sure the switch is running and that you have the right port
hemant@ubuntu:~/behavioral-model/tools$ 
```